### PR TITLE
Fix command runtime nix dep

### PR DESCRIPTION
**Pull request by luigy**
ob requires at least nix 2.0, but it was being wrapped with an older
version. This gets rid of having to manually set NIX_REMOTE for systems
running on 2.0

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -21,7 +21,7 @@ let
     git
     gitAndTools.hub
     nix-prefetch-git
-    nixStable
+    nixUnstable
```
Comment (luigy) [inline default.nix]: Or maybe it should not even manually wrap any `nix` in here since whatever version the user is running would be more correct to run against the `nix-daemon`

Comment (ElvishJerricco) [reply default.nix]: I agree with that. You should definitely be using the version they have installed, or else you risk miscommunication with the daemon

**Comment by 3noch** [comment]
Hm are you sure it's because `nixStable` is old? Opening `nix repl '<nixpkgs>'` and running `nixStable` gives me nix 2. I don't think `nixUnstable` is actually preferable since I *think* that refers to an unofficial release of nix? Also, since obelisk is pinned to a version of nixpkgs, this `nixStable` can only refer to the version of nix in the pin. If removing this fixes an issue, great, but I'm a bit suspicious of the diagnosis.

**Comment by 3noch** [comment]
Oh I just realized that the *pin* itself is still on nix 1.11! Yes that does seem problematic. *However* I still don't think using `nixUnstable` is the cure, but using `nixStable` from a nixpkgs that gives us nix 2.

File: default.nix (hunk 1 above)
Comment (3noch) [reply default.nix]: Hm I haven't encountered this directly. For example, `nix-shell -p nixStable2` on NixOS 17.09 (still using nix 1) works fine and even nested shells. However, I'm no expert on how the daemon works so I could see that being a problem.

Comment (ElvishJerricco) [reply default.nix]: Opening a nix-shell isn't exactly a comprehensive test of all the nix-daemon's capabilities :P I've had breakage in corner cases between 2.0.x bumps, so I'm cautious to use different versions in any scenario. There's nothing stopping them from making major breaking changes to the daemon's protocol in the future, as I don't think it's considered public API.

Comment (3noch) [reply default.nix]: Yeah if there's legitimate concern then I'm all for avoiding issues.

**Comment by ryantrinkle** [comment]
The real fix here is to get reflex-platform's reunification branch up to date asap; it's getting pretty close.
